src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -170,7 +170,7 @@ public function testCopyForOriginUrlsAndExistingLocalFileDefaultsToCopy()
         if (!\in_array('https', stream_get_wrappers())) {
             $this->markTestSkipped('"https" stream wrapper is not enabled.');
         }
-        $sourceFilePath = 'https://symfony.com/images/common/logo/logo_symfony_header.png';
+        $sourceFilePath = 'https://symfony.com/logos/symfony_black_02.png';
```

Comment:
```suggestion
        $sourceFilePath = 'https://symfony.com/favicon.ico';
```

< 4ko, and this URL is 99.9% guaranteed to not change soon 

🤷 

(just a suggestion passing by :) )